### PR TITLE
Add Halodao skin

### PR DESCRIPTION
**Pull request by mystbrent**
#### Hi! What is your PR about?

- [ ] Add images for your space
- [ ] Add custom domain for your space
- [ ] Add alias for migrate your space
- [x] Add or edit a skin
- ~Edit your space with GitHub?~ (deprecated) Please migrate your space to ENS https://docs.snapshot.page/spaces/migrate
- ~Create your space with GitHub~ (deprecated) Create your space with ENS instead https://docs.snapshot.page/spaces/create

### CODE REVIEW

File: skins/halodao.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  }
+
+  #topnav a {
+    color: white !important;
+  }
+
+  #topnav button {
+    background: linear-gradient(54.93deg, #15006D 12.16%, #15006D 33.28%, #5521B6 66.19%, #2DB7C4 93.15%);
+    border-radius: 53px;
+  }
```
Comment (gokaiorg) [inline skins/halodao.scss:22]: Please use only theme variables. No extra custom styles are allowed except on #topnav.
```suggestion
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

File: skins/halodao.scss (hunk 1 above)
Comment (mystbrent) [reply skins/halodao.scss]: I see copy that

**Comment by mystbrent** [comment]
Custom styles are removed on the latest commit changes

File: skins/halodao.scss [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  --link-color: #111111;
+  --heading-color: #FFFFFF;
+  --border-color: #d1d5da;
+  --header-bg: #471BB2;
+  --block-bg: white;
+
+  #topnav {
+    background-color: #FFFFFF;
+  }
+}
```
Comment (gokaiorg) [inline skins/halodao.scss:14]: The colors used in your skin make the interface unreadable, please change with others or remove the skin to use the default theme.
image

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Comment by gokaiorg** [comment]
Closing this PR for now, feel free to open it again once you've done the changes.

**Comment by mystbrent** [comment]
Added changes will open the PR

**Pull request closed without merging**